File: demo/springboot/demo/src/main/java/com/demo/domain/dto/http/resonse/OssUserDeleteAllResponse.java
package com.demo.domain.dto.http.resonse;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

@Data
public class OssUserDeleteAllResponse {
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private Long userId;
    private Integer totalCount;
    private Integer successCount;
    private Integer failCount;
    private String message;
}

File: demo/app/app/src/main/java/com/vectordemo/domain/dto/http/response/OssUserDeleteAllResponse.kt
package com.vectordemo.domain.dto.http.response

import com.google.gson.annotations.JsonAdapter
import com.vectordemo.utils.json.WireUserIdJsonDeserializer

data class OssUserDeleteAllResponse(
    @JsonAdapter(WireUserIdJsonDeserializer::class)
    val userId: String? = null,
    val totalCount: Int? = null,
    val successCount: Int? = null,
    val failCount: Int? = null,
    val message: String? = null
)
